Sources_v3/StreamChatUI/ChatChannel/MessageComposerCommandCollectionViewCell.swift

```diff
@@ -41,9 +41,11 @@ open class MessageComposerCommandCellView<ExtraData: ExtraDataTypes>: View, UICo
     }
 
     override open func updateContent() {
-        commandNameSubtitleLabel.text = content?.subtitle
-        commandNameLabel.text = content?.title
-        commandImageView.image = content?.commandImage
+        guard let command = command else { return }
+        commandNameSubtitleLabel.text = "/\(command.name) \(command.args)"
+        commandNameLabel.text = command.name.firstUppercased
+        commandImageView.image = uiConfig.messageComposer.commandIcons[command.name.lowercased()] ??
+            uiConfig.messageComposer.commandIcons["fallback"]
```

Comment:
nit: The fallback icon probably shouldn't come from the dictionary but rather be some kind of constant. This will allow you to do `config.commandIcons: ["command": icon, ....]` without having to worry about ovrriding `fallback`

Reply:
Thanks, makes sense! 
3af5238